crux-core/src/crux/codec.clj

```diff
@@ -94,6 +94,7 @@
   (mem/limit-buffer to id-size))
 
 (def ^:dynamic ^:private *sort-unordered-colls* false)
+(alter-var-root #'nippy/*freeze-serializable-allowlist* (fn [_] nippy/*thaw-serializable-allowlist*))
```

Comment:
Would this change Nippy for all uses of Nippy, even when Crux embeddable in a users application?

Reply:
That's a very good question - in short, I'm not sure, but I think you may well be correct that it would have that behaviour. I'll check that out - I suppose alternatively we could make this a `binding` call and make a function to do it within `crux.io`, ensure that any calls we have to use freeze/thaw do the binding.

Reply:
yes, trying to avoid an alter-var-root with a binding would be preferable.